msal/src/Microsoft.Identity.Client/UIParent.cs

```diff
@@ -52,6 +52,7 @@ public UIParent()
 #if iOS
         public UIParent(bool useEmbeddedWebview)
         {
+            CoreUIParent = new CoreUIParent();
```

Comment:
>CoreUIParent = new CoreUIParent(); [](start = 12, length = 34)

call this after the constructor as that will do what the default constructor does

Reply:
done. thanks.

---
In reply to: [189374400](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1057#discussion_r189374400) [](ancestors = 189374400)